.github/workflows/ci-backend.yml

```
@@ -283,14 +284,87 @@ jobs:
+      - name: Generate Current OpenAPI Spec
+        env:
+          ASPNETCORE_ENVIRONMENT: Testing
+        run: |
+          mkdir -p api-current
+          swagger tofile \
+            --output api-current/api-spec.json \
+            src/Bootstrapper/MeAjudaAi.ApiService/bin/Release/net10.0/MeAjudaAi.ApiService.dll \
+            v1
+
+      - name: Checkout Base Branch Spec
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)